src/components/Support/DonationInstructions.tsx

```diff
@@ -0,0 +1,38 @@
+import { cn } from "@/utils";
+import { Heading } from "../shared/Heading/Heading";
+
+interface Props {
+  includeHeader?: boolean;
+}
+
+export const DonationInstructions = ({ includeHeader }: Props) => {
+  return (
+    <>
+      {includeHeader && (
+        <Heading variant="h4" color="green" contrast="black">
```

Comment:
I believe we could just remove this `includeHeader` prop. And move this `Heading` to the consuming component of `<TaxSection />`

Reply:
Like in [this PR](https://github.com/Delicadency/AK1944/pull/64) here: https://github.com/Delicadency/AK1944/pull/64/commits/925e6d012b2f43195849e94532c3e090af67edc5